src/components/ResultDetail/FeesInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { FeePreference } from "../../types";
+
+function FeesInfo({
+  fees,
+  isCondensed,
+}: {
+  fees: { [key in FeePreference]: boolean };
+  isCondensed?: boolean;
+}) {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      {Object.entries(fees)
+        .filter(([_, val]) => !!val)
+        .map(([key], idx, arr) => (
+          <React.Fragment key={key}>
+            {t(`feesValues${key}`)}
+            {!isCondensed && key === "PrivateInsurance" && "*"}
+            {idx < arr.length - 1 ? ", " : ""}
+          </React.Fragment>
+        ))}
+      {!isCondensed && fees.PrivateInsurance && (
+        <div className="margin-top-05">*{t(`privateInsuranceNote`)}</div>
```

Comment:
should this font be slightly smaller ? my eye initially overwhelmed by the note text looking the same as the actual info

Reply:
Agree @jlhogan. @cathydeng can we knock the insurance note down to .75rem and see now that looks? May need to adjust space above too, but I'd like to see it smaller first.